src/components/Layout/Header/Header.module.css

```
@@ -0,0 +1,9 @@
+.header {
+  position: fixed; /* 화면에 고정시 */
+  top: 0;
+  left: 0;
+  width: 100%;
+  zindex: 1000;
```

Stylelint (17.3.0)

[error] Unexpected unknown property "zindex" (property-no-unknown)

(property-no-unknown)